### ISSUE TITLE
Add support for entering special characters like Emojis

### ISSUE THREAD

**Issue by DraconicNEO** [waffle-stomper/Ghostwriter#48]
**Is your feature request related to a problem? Please describe.**
In the mod there isn't an option to select special characters like emojis or symbols which makes it very hard to enter these into books.

**Describe the solution you'd like**
Implement this feature into ghostwritter

**Describe alternatives you've considered**
N/A

**Additional context**

**Comment by waffle-stomper** [comment]
Can you enter these symbols into vanilla books? I was under the impression they were limited to ASCII, but I haven't tried it so I could be wrong.

**Comment by DraconicNEO** [comment]
> Can you enter these symbols into vanilla books? I was under the impression they were limited to ASCII, but I haven't tried it so I could be wrong.

Yeah, I've done it by copy-pasting and also I think another mod Stendhal supports this function.
Pretty sure modern versions of the game are unicode since even emojis work, though they're monochrome and not all of them show up (flag emojis are one example).
